Explain Ctrl + ScrollWheel can change Grid Floor (#11919)
godotengine/godot-docs: tutorials/3d/using_gridmaps.rst

  that will be painted on the Z-axis. This will also rotate selected areas if they
  are being moved.

> - **Change Grid Floor**: Adjusts what floor is currently being worked on, can be

Reviewer: ```suggestion
- **Change Grid Floor**: Adjusts what floor is currently being worked on. Can be
```

> - **Change Grid Floor**: Adjusts what floor is currently being worked on, can be

Reviewer: ```suggestion
- **Change Grid Floor**: Adjusts what floor is currently being worked on. Can be
```

>   changed with the arrows or typing or :kbd:`Ctrl + ScrollWheel`

Reviewer: ```suggestion
  changed with the arrows, typing a value in the field, or :kbd:`Ctrl + Mouse wheel`.
```
